web3/_utils/request.py

```diff
@@ -63,56 +64,88 @@ def cache_session(endpoint_uri: URI, session: requests.Session) -> None:
     _session_cache[cache_key] = session
 
 
-async def cache_async_session(endpoint_uri: URI, session: ClientSession) -> None:
-    cache_key = generate_cache_key(endpoint_uri)
-    with _async_session_cache_lock:
-        evicted_items = _async_session_cache.cache(cache_key, session)
-        if evicted_items is not None:
-            for key, session in evicted_items.items():
-                await session.close()
-
-
-def _remove_session(key: str, session: requests.Session) -> None:
+def _remove_session(_key: str, session: requests.Session) -> None:
     session.close()
 
 
 _session_cache = lru.LRU(8, callback=_remove_session)
-_async_session_cache_lock = threading.Lock()
-_async_session_cache = SessionCache(size=20)
 
 
-def _get_session(endpoint_uri: URI) -> requests.Session:
+def get_session(endpoint_uri: URI) -> requests.Session:
```

Comment:
total nit: did you mean to change this from `_get_session` to `get_session`?

Reply:
Yeah. I ised this in another internal file... I thought the fact that it's already in `_utils` should detract from anyone using this (technically) and thought it made more sense since it's no longer only used internally in this file, but now also in my mocking methods in module testing utils.